src/stdlib/mezmo_redact.rs

```diff
@@ -113,6 +114,13 @@ impl Function for MezmoRedact {
             })
             .collect::<std::result::Result<Vec<FilterWithRedactor>, _>>()?;
 
+        // The phone number pattern may match parts of credit cards (no delimiters)
+        // To reduce the blast radius, move the phone filter to the last item
+        filters.sort_by(|item1, item2| match (&item1.filter, &item2.filter) {
+            (&Filter::PhoneNumber, _) => Ordering::Equal,
```

Comment:
does this need to be `Ordering::Greater`?

Reply:
Yes absolutely